modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -620,10 +620,18 @@
 			<param index="3" name="transfer_channel" type="int" default="0" />
 			<description>
 				Mark the following method for remote procedure calls. See [url=$DOCS_URL/tutorials/networking/high_level_multiplayer.html]High-level multiplayer[/url].
-				The accepted values:
-				- for [param mode] are [code]"any_peer"[/code] or [code]"authority"[/code];
-				- for [param sync] are [code]"call_remote"[/code] or [code]"call_local"[/code];
-				- and for [param transfer_mode] are [code]"unreliable"[/code], [code]"unreliable_ordered"[/code], or [code]"reliable"[/code].
+
```

Comment:
And empty line must definitely go, we don't do that because it also creates an entire paragraph.